independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -256,66 +286,131 @@
+                  "description": "Last edited by",
+                  "title": "Last edited by"
+                },
                 "has_children": {
                   "type": "boolean",
                   "description": "Whether or not the block has children blocks nested within it.",
-                  "title": "Has children"
-                },
-                "type": {
-                  "type": "string",
-                  "description": "Type of block. Possible values",
-                  "title": "Type"
+                  "title": "Has children",
+                  "enum": [
+                    true,
+                    false
+                  ]
                 },
                 "archived": {
                   "type": "boolean",
                   "description": "The archived status of the block.",
-                  "title": "archived"
-                },
-                "content": {
-                  "type": "string",
-                  "description": "Updates the content for the specified block_id based on the block type.",
-                  "title": "Content"
+                  "title": "Archived",
+                  "enum": [
+                    true,
+                    false
+                  ]
                 },
-                "plain_text": {
+                "type": {
                   "type": "string",
-                  "description": "Plain text in the paragraph block plain_text",
-                  "title": "Plain text"
+                  "description": "An object containing a type-specific block information",
+                  "title": "Type"
                 }
               }
             }
           }
         },
-        "summary": "Update a block (to-do)",
-        "description": "Updates the content for the specified block_id based on the block type to-do.",
-        "operationId": "UpdateaBlock",
+        "summary": "Delete a block",
+        "operationId": "Deleteablock",
+        "description": "Sets a Block object, including page blocks, to archived: true using the ID specified. Note: in the Notion UI application, this moves the block to the \"Trash\" where it can still be accessed and restored.",
         "parameters": [
           {
-            "name": "block_id",
+            "name": "blockId",
             "in": "path",
             "required": true,
             "type": "string",
             "description": "Identifier for a Notion block",
-            "x-ms-summary": "Block id",
-            "x-ms-url-encoding":"single"
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Block ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Notion-Version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "default": "2022-06-28"
+          }
+        ]
+      },
+      "patch": {
+        "responses": {
+          "default": {
```

Comment:
@sekharmalla Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
Thanks, @sriyen-msft, 

I have updated the Default response with the Respective response codes. and most of the actions in this swagger have dynamic responses hence not defining.

Thanks